docs/content/docs/plugins/writing-a-vendure-plugin.md

```diff
@@ -38,6 +38,9 @@ import { VendurePlugin } from '@vendure/core';
 export class RandomCatPlugin {}
 ```
 
+Changing the database schema by `CustomFields` configuration requires a synchronized database. See [Customizing Models With Custom Fields](https://www.vendure.io/docs/developer-guide/customizing-models/)
```

Comment:
Could you make this into a relref link, so that the domain is not hard-coded? 

```
({{< relref "customizing-models" >}})
```

Just make sure to test it out in the Hugo build to ensure it works as expected.

Reply:
![image](https://user-images.githubusercontent.com/19676051/107646106-df696100-6c79-11eb-9c41-5d4d10273487.png)

Implemented and tested locally.